apple/internal/ios_rules.bzl

```diff
@@ -680,6 +665,25 @@ def _ios_framework_impl(ctx):
         res_attrs = ["resources"],
     )
 
+    extra_linkopts = [
+        "-dynamiclib",
+        "-Wl,-install_name,@rpath/{name}{extension}/{name}".format(
```

Comment:
I used this version to work around https://github.com/bazelbuild/bazel/blob/1c3a2456c95fd19974a5b2bd33c5ebdb2b2277e4/src/main/java/com/google/devtools/build/lib/rules/cpp/LinkCommandLine.java#L295 causing an issue. Think this is bug worthy? One nice thing about doing it this way: it's the same argument that was being sent before.

Reply:
Thanks. Yes I think we should report it.

Reply:
https://github.com/bazelbuild/bazel/issues/14316